room/compiler-processing-testing/src/main/java/androidx/room/compiler/processing/util/ProcessorTestExt.kt

```diff
@@ -115,12 +115,14 @@ fun runProcessorTest(
     sources: List<Source> = emptyList(),
     classpath: List<File> = emptyList(),
     options: Map<String, String> = emptyMap(),
+    kotlincArguments: List<String> = emptyList(),
```

Comment:
sure. I didn't end up using this since I couldn't get multi-platform testing to work, but might as well keep it in if you think it's useful (otherwise I can remove it)